mkdocs.yml

```diff
@@ -3,7 +3,7 @@ site_description: Documentation for the Lottie animation format
 extra:
     lottie_player_version: 5.12.2
 theme:
-    name: cinder
```

Comment:
If cinder is removed, it should be removed from requirements.txt as well

Reply:
brought back the theme with the updated styles